kafka-merge-pr.py

```diff
@@ -70,7 +70,7 @@
 
 DEV_BRANCH_NAME = "trunk"
 
-DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "2.1.0")
+DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "2.1.1")
```

Comment:
This one is not needed since this tool should always be executed from trunk.

Reply:
I agree it doesn't need to be, but it seems like it should be for consistency, and to avoid confusion.